### ISSUE TITLE
Setuptools causes issues

### ISSUE THREAD

**Issue by craig-reahl** [reahl/reahl#271]
When running tests or serving a site, with  setuptools > 40.7.3 installed:

**Linux OS based virtualenv - deprecation warning**:

```
py.warnings:warnings.py:99 /home/vagrant/.virtualenvs/python3.6/lib/python3.6/site-packages/pkg_resources/__init__.py:1146: DeprecationWarning: Use of .. or absolute path in a resource path is not allowed and will raise exceptions in a future release.
```

**Windows OS based - ValueError**: 

```
File "c:\users\hendr\projects\reahl_env\lib\site-packages\pkg_resources\__init__.py", line 1559, in _validate_resource_path
    raise ValueError(msg)
ValueError: Use of .. or absolute path in a resource path is not allowed.
```

This seems to have been a problem for Windows OS after setuptools==40.7.3

**Workaround** - install last lenient setuptools:

`pip install setuptools==40.7.3`

**Comment by craig-reahl** [comment]
It seems that on Windows this is still causing an issue. Although work has been done on this already, another issue has been found, and appreciatively reported by one of our Windows users.

**Comment by IwanVosloo** [comment]
Fixed in #304
